### DISCUSSION TITLE
Can I use multiple remote url on one repository?

### DISCUSSION THREAD

**Discussion by VegetableDev0701** [community/community#114665] [New to GitHub]
### Select Topic Area

Question

### Body

Hi, I am newbie to github.
I am now using One repo with multiple remote url.
but fetch from one remote, push to another remote every times.
is there any easy work fetching from one to push to another automatically ?
Not with Fork.
Thanks.

**Comment by ettaboyle** [comment]
Thanks for posting in the GitHub Community, @VegetableDev0701! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions). This is a great question for the New to GitHub category. I've gone ahead and moved it for you. Good luck!
